test/gurka/test_gtfs.cc

```diff
@@ -398,6 +433,12 @@ TEST(GtfsExample, MakeTile) {
   auto station_two_ll = layout.find("2");
   auto station_three_ll = layout.find("3");
 
+  auto dt = "2023-02-27T05:50";
+  auto dt_date = DateTime::get_formatted_date(dt);
+  auto dt_date_days = DateTime::days_from_pivot_date(dt_date);
+  auto dt_dow = DateTime::day_of_week_mask(dt);
+  bool date_before_tile = false;
```

Comment:
only added these to test the `tile->GetNextDeparture`. think I'll remove it again (or be smarter around choosing a date, see above comment)